src/briefcase/ios.py

```diff
@@ -129,8 +129,18 @@ def set_device_target(self):
 
             data = json.loads(output)
 
+            def clean_label_output(label):
+                if label.startswith('com.apple.CoreSimulator.SimRuntime.iOS'):
+                    return label[-8:-2]
+                else:
+                    return label
+
             if self.os_version is None:
-                os_list = [label for label in data['devices'] if label.startswith('iOS')]
+                os_list = []
+                for label in data['devices']:
+                    if label.startswith('iOS') or label.startswith('com.apple.CoreSimulator.SimRuntime.iOS'):
+                        os_list.append(clean_label_output(label))
```

Comment:
Rather than try to catch both 'startswith' cases in a single if clause, and then have an inner if - why not handle this with a single if/elif pair?
```
if startswith iOS
    append label
elif startswith com.apple...
    append trimmed label
```
That also removes the need for an inline function.

Reply:
@magic-lantern @freakboy3742 
Apologies with late reply. Seems I have some Python and 'zsh' related issues at my home computer. I realized this when I've just tried doing some of the patching now. 

So, I wrote the patch but couldn't do the testing. Is it possible to get some help testing this out with some different configuration (macOS versions etc)?

Reply:
I'm running macOS 10.14.4 with Xcode 10.2.1. The proposed changes to ios.py do resolve the problem of "No iOS device simulators found" and correctly start an iOS simulator with the selected iOS version and hardware.

I am getting some new/additional errors though. My errors I'm receiving appear to be the same as mentioned in #245, so I believe they are separate from this pull request.